src/utils/archanova.js

```diff
@@ -487,3 +500,27 @@ export const buildEnsMigrationRawTransactions = async (accounts: Account[], wall
       return data;
     });
 };
+
+/**
+ * Checks accounts for migrated ENS in following order:
+ * 1. First checks Etherspot account in case ENS already migrated and was updated on Etherspot back-end.
+ * 2. If there's no ENS yet on Etherspot account or transaction is pending then it checks on Archanova
+ */
+export const getMigratedEnsName = (accounts: Account[]): string => {
```

Comment:
Looking at the signature or implementation there is nothing specific to archanova (as it also handles etherspot) nor to migration.  I wonder if it would make sens to rename it as `getPrimaryEnsName` and move it to accounts?

Reply:
Makes sense to move it, thanks. Not sure about renaming tho, I don't want this method to be used elsewhere besides migration and with `getPrimaryEnsName` it's easy to fall into trap, hope it makes sense.